source/ui/acquaman/AMGenericStepScanConfigurationView.cpp

```diff
@@ -106,8 +106,15 @@ AMGenericStepScanConfigurationView::AMGenericStepScanConfigurationView(AMGeneric
 
 	positionsBox->setLayout(positionLayout);
 
-	onAxisControlChoice1Changed();
-	onAxisControlChoice2Changed();
+	// Set initial spinbox values in cases where we don't yet have axes:
+	if(configuration_->scanAxes().count() == 0) {
```

Comment:
I've tested the same change on the SGMUpgrade branch, and it behaves correctly with new scans with no controls yet selected, loaded 1d scan configurations and loaded 2d scan configurations.